docs/microbit.rst

```diff
@@ -62,6 +62,7 @@ Classes
     :maxdepth: 1
 
     image.rst
+    sound
 
 
 Modules
```

Comment:
Is it in `microbit` for V2 only? If so, should I indicate that like `audio V2`

Reply:
Yes, good point, it is inside microbit only on v2

Reply:
It's not easy to do this due to the way `toctree` works. It will not let you embolden text in links and you can't use relative links either, so sound would really have to link to a page ie the general micropython API page or we just don't include sound (or I make a page for it). 

```
    Audio V2 <audio.rst>
    image.rst
    Sound <microbit_micropython_api.rst>
````
renders as
![image](https://user-images.githubusercontent.com/31242877/100474543-c3bdd800-30d8-11eb-8ffd-f12fd493afba.png)

Reply:
I didn't quite understand that sorry.

Based on your example, wouldn't we be able to do this?

```
Modules
.. toctree::
    :maxdepth: 1

    Audio (V2 only) <audio.rst>
    accelerometer.rst
    ....
```

Reply:
I just noticed the example you provided is already in the PR.

I think this works, right?
Only thing is to move the Audio entry from classes to modules.

Reply:
Yes, i think it's okay as it is. I have moved it in https://github.com/bbcmicrobit/micropython/pull/692/commits/bbe4d057d132c57bab29873d8b9bc40279787968